src/main/java/ch/njol/skript/expressions/ExprVersionString.java

```diff
@@ -0,0 +1,115 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.ScriptLoader;
+import ch.njol.util.coll.CollectionUtils;
+import ch.njol.util.Kleenean;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.event.server.ServerListPingEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Version String")
+@Description({"The text to show if protocol version of the server doesn't match with protocol version of the client. " +
+		"You can check the <a href='#ExprProtocolVersion'>protocol version</a> expression for more information about this.",
+		"This can be set in a <a href='events.html#server_list_ping'>server list ping</a> event only.",
+		"",
+		"Requires PaperSpigot 1.12.2+."})
+@Examples({"on server list ping:",
+		"	set the protocol version to 0 # 13w41a (1.7), so it will show the version string always",
+		"	set the version string to \"<light green>Version: <orange>%minecraft version%\""})
+@Since("INSERT VERSION")
+public class ExprVersionString extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprVersionString.class, String.class, ExpressionType.SIMPLE, "[the] [(shown|custom|fake)] version [(string|text)]");
```

Comment:
what is the point of allowing fake in the syntax if it may still return the "real" version string?